sympy/printing/llvmjitcode.py

```diff
@@ -0,0 +1,192 @@
+from __future__ import print_function, division
+
+'''
+Use llvmlite to create executable functions from Sympy expressions
+
+This module requires llvmlite (https://github.com/numba/llvmlite).
+'''
+
+import ctypes
+
+from sympy.external import import_module
+from sympy.printing.printer import Printer
+from sympy import S
+from sympy.utilities.decorator import doctest_depends_on
+
+llvmlite = import_module('llvmlite')
+if llvmlite:
```

Comment:
Yes, that is what happens. You need to raise an import error in the Printer's __init__ function based on `if llvmlite`.